code/modules/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -90,7 +90,7 @@
 	progression_minimum = 15 MINUTES
 	progression_maximum = 60 MINUTES
 	progression_reward = list(8 MINUTES, 12 MINUTES)
-	telecrystal_reward = list(2, 3)
+	telecrystal_reward = list(4)
```

Comment:
Same thing here, this was fine to be kept as list(2, 3)
The alive bonus already compensates a lot of TC for pulling this off.